brainscore_vision/model_helpers/activations/core.py

```diff
@@ -578,15 +579,10 @@ def attach_stimulus_set_meta(assembly, stimulus_set, number_of_trials: int, requ
     assert (np.array(assembly_paths) == np.array(repeated_stimulus_paths)).all()
     repeated_stimulus_ids = np.repeat(stimulus_set['stimulus_id'].values, replication_factor)
 
-    if replication_factor > 1:
```

Comment:
Removed since assembly is extended automatically by `ActivationsExtractorHelper`. Added a new test in `test___init__.py` to catch any future errors related to this.